Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/TwoHandManipulatableMode.cs

```diff
@@ -1,17 +1,19 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
+using System;
 
 namespace HoloToolkit.Unity.InputModule.Utilities.Interactions
 {
     /// <summary>
     /// Sorting type for collections
     /// </summary>
+    [Flags]
     public enum TwoHandManipulatableMode
     {
-        Scale,
-        Rotate,
-        MoveScale,
-        RotateScale,
-        MoveRotateScale
+        Scale = (1 << 0),
+        Rotate = (1 << 1),
+        MoveScale = (1 << 2),
+        RotateScale = (1 << 3),
+        MoveRotateScale = (1 << 4),
```

Comment:
I'm suggesting doing something like this:

```
[Flags]
     public enum TwoHandManipulatableMode
{
    None = 1<<0,
    Move = 1<<1,
    Rotate = 1<<2,
    Scale = 1<<3,
}
```

Then in the switch statement, it'll actually get updated to something else that handles it correctly.